cypress/integration/channel/header_spec.js

```diff
@@ -35,4 +35,25 @@ describe('Header', () => {
         // * Check if channel header description has ellipsis
         cy.get('#channelHeaderDescription').ellipsis(true);
     });
+
+    it('MM-14730 - Validate that channel header displays quote correctly', () => {
```

Comment:
Please change using test key which we're using to track written code against test cases.
```suggestion
    it('CS14730 - Channel Header : Markdown quote', () => {
```

Reply:
👍